2018 November Planning Session Notes, Update Lead Roles

dasanchez opened the pull request.

dasanchez committed:
Notes for 2018 November Planning Session

Shrinks99 commented:
https://github.com/tomeshnet/documents/blob/master/governance/coordination-structure.md Should be updated accordingly.

- People being around for one cycle before assigned lead roles
- Updated lead list

Anything else I'm missing?  If we want to do that in a separate PR that's fine and this looks good.  If you want those bundled with this one I'd be cool with that.  Either way will approve once that's decided upon.

dasanchez committed:
Update coordination-structure.md

- Updated working group leads
- Prior involvement required for lead roles.

dasanchez commented:
Thanks @Shrinks99! It makes sense to me to revise the coordination structure doc at this time.

Shrinks99 committed:
Update coordination-structure.md

Updated the collaboration tools with small changes for links and better wording.

Shrinks99 reviewed (approved):
Looks good to me!

benhylau reviewed (commented):
Minor

benhylau commented on meeting_notes/20181122_planning-meeting-notes.md:
```diff
@@ -0,0 +1,147 @@
+
+- Adding SSB (patchfoo) to the prototype stack
+- See v0.4 roadmap: Yggdrasil + Install Profiles in prototype
+- Contact at Alpha Networks if anyone's interested in buying from them
```
```suggestion
- Contact at [Alfa Network](https://www.alfa.com.tw) if anyone's interested in buying from them
```

benhylau committed:
Update meeting_notes/20181122_planning-meeting-notes.md

It's _Alfa network_. Thanks @benhylau!

Co-Authored-By: dasanchez <dante.a.sanchez@gmail.com>
